src/fromager/bootstrapper.py

```diff
@@ -323,6 +323,10 @@ def _download_wheel_from_cache(
             wheelfile_name = pathlib.Path(urlparse(wheel_url).path)
             pbi = self.ctx.package_build_info(req)
             expected_build_tag = pbi.build_tag(resolved_version)
+            # Log the expected build tag for debugging
+            if expected_build_tag:
```

Comment:
The goal is to log more complete debug information about the build tag, so that someone looking at the log can understand why fromager took whatever steps it did. So we want a message even if there is no build tag.